log/log.go

```diff
@@ -104,6 +105,16 @@ var GlobalLogger = func() insolar.Logger {
 		panic("couldn't initialize global logger with default config")
 	}
 
+	if logger.Is(insolar.DebugLevel) {
+		go func() {
+			for {
```

Comment:
my point was
`
ticker := time.NewTicker(time.Second)
	for range ticker.C {
		fmt.Println("Hello, playground")
	}
`

Reply:
it's cleaner and more "go way"

Reply:
but it's not that accurate